internal/beater/config/config_test.go

```diff
@@ -362,7 +362,6 @@ func TestUnpackConfig(t *testing.T) {
 						ESConfig:              elasticsearch.DefaultConfig(),
 						Interval:              1 * time.Minute,
 						IngestRateDecayFactor: 0.25,
-						StorageGCInterval:     5 * time.Minute,
```

Comment:
[to reviewer] I tested with apm-server.yml that an extra options are silently ignored, so a config removal should be fine